subcommand/server-acl-init/command.go

```diff
@@ -50,6 +50,8 @@ type Command struct {
 	flagCreateSnapshotAgentToken bool
 
 	flagCreateMeshGatewayToken bool
+	flagIngressGateways        []string
```

Comment:
What about flagIngressGatewayNames? Looks like we follow the convention of matching these variables with the flags.

Reply:
Updated.